### ISSUE TITLE
Can I parameter share batch normalization layers?

### ISSUE THREAD

**Issue by eldercrow** [dmlc/mxnet#5213]
I know how to share parameters between conv and fc layers, by first declaring mx.sym.Variable's then giving them into symbols to be parameter shared. But I wonder if I can do a similar stuff to BatchNorm layers, since I also want to share (or at least copy from pre-trained one) data mean and variances, which are defined as outputs of BN layers. 
If parameter sharing is not possible, then at least I want to copy mean and variances from one layer to other layers before training.

**Comment by piiswrong** [comment]
It is computationally impossible to share per-batch mean and var between sequential layers because there are dependencies. If the layers are in parallel streams then you can concatenate them before feeding to batchnorm
